mkdocs.yml

```diff
@@ -121,7 +124,12 @@ markdown_extensions:
       anchor_linenums: true
   - pymdownx.inlinehilite
   - pymdownx.snippets
-  - pymdownx.superfences
+  - pymdownx.superfences:
+      custom_fences:
+        - name: mermaid
+          class: mermaid
+          format: !!python/name:mermaid2.fence_mermaid_custom
```

Comment:
Pre-commit is complaining about this line, and I'm not sure why! Anything we can do about that?

Reply:
Looks like we can add `--unsafe` arg to pre-commit to get by: https://github.com/pre-commit/pre-commit-hooks#check-yaml

Reply:
Yeah, I don't think we should change the mkdocs config. We should change the pre-commit config! See the link above :)